MiniDumper/MiniDumper.cs

```diff
@@ -217,9 +310,15 @@ public int NumberOfDumpsTaken
             get { return numberOfDumpsTaken; }
         }
 
+        public bool IsProcessTerminated { get => isProcessTerminated; }
+
+
+
         public void Dispose()
         {
             target.Dispose();
+            hProcess.Dispose();
```

Comment:
Please check if hProcess is not null - we don't use it in other situations than memory commits checks and it throws NullPointerException.